e2e/cli/command/run.go

```diff
@@ -192,10 +192,10 @@ func (c *Run) runTest(opts *runOpts) (*TestReport, error) {
 	cmd := exec.Command(goBin, opts.goArgs()...)
 	cmd.Env = opts.goEnv()
 	out, err := cmd.StdoutPipe()
-	defer out.Close()
```

Comment:
I'm looking at the docs for [`os/exec.Cmd.StdoutPipe`](https://golang.org/pkg/os/exec/#Cmd.StdoutPipe) and I'm wondering if we should be calling `Close` at all in this function, rather than `Wait`.